everware/tests/test_repourl_parser.py

```diff
@@ -0,0 +1,40 @@
+from .. import git_executor
+
+def test_parser():
+    tests = [
+        (
+            'git://github.com/aladagemre/django-notification.git@2927346f4c513a217ac8ad076e494dd1adbf70e1',
```

Comment:
I think we should not let people submit `git://` -> give an error message in the UI when they do and raise an exception if they sneak it past the JS.